centreon/packages/ui/src/Colors/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { makeStyles } from 'tss-react/mui';
+import { includes, toPairs } from 'ramda';
+
+import { Stack, Box, useTheme, Typography, Divider, Grid } from '@mui/material';
+
+const useStyles = makeStyles()((theme) => ({
+  divider: {
+    margin: theme.spacing(2)
+  },
+  headerContainer: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  itemContainer: {
+    borderRadius: '5px',
+    margin: theme.spacing(2),
+    padding: theme.spacing(1),
+    textAlign: 'center'
+  },
+  itemContainerText: {
+    border: '0.3px solid',
+    borderRadius: '5px',
+    margin: theme.spacing(2),
+    padding: theme.spacing(1),
+    textAlign: 'center'
+  }
+}));
+
+interface ContainerDescriptionProps {
+  containerTitle: string;
+  keyTheme: string;
+}
+const ContainerDescription = ({
+  keyTheme,
+  containerTitle
+}: ContainerDescriptionProps): JSX.Element => {
+  const { classes } = useStyles();
+  const { palette } = useTheme();
+
+  const listKeyToRemove = [
+    'contrastText',
+    'activatedOpacity',
+    'focusOpacity',
+    'hoverOpacity',
+    'selectedOpacity',
+    'disabledOpacity'
+  ];
+
+  return (
+    <Box sx={{ width: '100%' }}>
+      <Typography variant="h4">{containerTitle}</Typography>
+      <Grid container columnSpacing={{ md: 3, sm: 2, xs: 1 }} rowSpacing={5}>
+        {toPairs(palette[keyTheme]).map(
+          ([key, value]) =>
+            !includes(key, listKeyToRemove) && (
+              <Grid item key={key} xs={6}>
+                <div className={classes.headerContainer}>
+                  <Typography variant="h6">{key}</Typography>
+                  <Typography variant="button">{value}</Typography>
+                </div>
+
+                <Box
+                  sx={{
+                    backgroundColor: value,
+                    borderRadius: '5px',
```

Comment:
```suggestion
                    borderRadius: 1,
```
As the border-radius is defined in the MUI's theme as 4px, you can use a number in order to set the border-radius, using the sx prop. 1 means 4px, 2 means 8px, etc...